src/test/java/net/sf/jabref/logic/formatter/casechanger/TitleCaseFormatterTest.java

```diff
@@ -17,20 +17,47 @@ public void setUp() {
     }
 
     @Test
-    public void test() {
+    public void upperEachFirst() {
         Assert.assertEquals("Upper Each First", formatter.format("upper each first"));
+    }
+
+    @Test
+    public void anUpperEachFirstAnd() {
         Assert.assertEquals("An Upper Each First And", formatter.format("an upper each first and"));
+    }
+
+    @Test
+    public void anUpperEachOfTheAndFirstAnd1(){
```

Comment:
I don't like the tests names too much since they don't describe the behavior which is expected but the input.
What about `prepositionIsLeftUntouched` and `theAfterColonGetsCapitalized` ?